taskt/Core/Automation/Commands/Excel Commands/ExcelCreateApplicationCommand.cs

```diff
@@ -2,67 +2,73 @@
 using System.Collections.Generic;
 using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using taskt.Core.Utilities.CommonUtilities;
+using Application = Microsoft.Office.Interop.Excel.Application;
 
 namespace taskt.Core.Automation.Commands
 {
-
-
-
-
     [Serializable]
-    [Attributes.ClassAttributes.Group("Excel Commands")]
-    [Attributes.ClassAttributes.Description("This command opens the Excel Application.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to launch a new instance of Excel.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements Excel Interop to achieve automation.")]
+    [Group("Excel Commands")]
+    [Description("This command creates an Excel Instance.")]
+
     public class ExcelCreateApplicationCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Enter the instance name")]
-        [Attributes.PropertyAttributes.InputSpecification("Signifies a unique name that will represemt the application instance.  This unique name allows you to refer to the instance by name in future commands, ensuring that the commands you specify run against the correct application.")]
-        [Attributes.PropertyAttributes.SampleUsage("**myInstance** or **excelInstance**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyDescription("Excel Instance Name")]
+        [InputSpecification("Signifies a unique name that will represent the application instance. " +
+            "This unique name allows you to refer to the instance by name in future commands, " +
+            "ensuring that the commands you specify run against the correct application.")]
```

Comment:
You can simply keep the input specification short like "Enter a unique name that will represent the excel application instance."

You can put the extra information in Remarks. Also remove the "allows you to" part from the text.

Reply:
Okay, I'll split this up, but I left you a similar comment in your IE browser commands so please check that they match

Reply:
I think this particular use of allows is necessary though